### DISCUSSION TITLE
groupBy not being recognized in TS DataGrid or TreeDataGrid

### DISCUSSION THREAD

**Discussion by conklin20** [Comcast/react-data-grid#3620] [Q&A]
I've searched many of the documents and issues looking for something along the same lines but haven't found anything, which makes me think it's something stupid on my end. For the life of me I can't get my DataGrid component to recognize the groupBy,  rowGrouper, expandedGroupIds, or onExpandedGroupIdsChange props. 
![image](https://github.com/user-attachments/assets/dce433a6-2bd1-453d-886f-bdf5a6e3bc4a)
`
    return (
        <DataGrid
            rows={rows || []}
            columns={cols}
            defaultColumnOptions={{
                sortable: true,
                resizable: true,
            }}
            onRowsChange={handleSetRows}
            sortColumns={handleSort}
            onSortColumnsChange={handleSetSorted}
            // rowKeyGetter={(row: any) => row.id}
            groupBy={groupBy}
            rowGrouper={rowGrouper}
            expandedGroupIds={expandedGroupIds}
            onExpandedGroupIdsChange={onExpandedGroupIdsChange}
            style={{ minHeight: '85%', width: '100%' }}
        />
    );
`

**Comment by daryazherya** [comment]
@conklin20, Did you find answer for your question? I have the same problem

**Reply by conklin20** [reply]
Unfortunately I have not.
